apps/stats/management/commands/__init__.py

```diff
@@ -92,3 +101,24 @@ def get_date_from_file(filepath, sep):
             return line.split(sep)[0]
         except IndexError:
             return None
+
+
+def stats_from_model(model):
+    """Return the stats from the model ready to write to a file."""
+    data = model_to_dict(model)
+    del data['date']  # Already in the filename.
+    del data['addon']  # Already in the file's folder name.
+    del data['id']  # No need for the model's ID at all (eg: UpdateCount).
+    return json.dumps(data)
+
+
+def save_stats_to_file(model):
+    """Save the given model to a file on the disc."""
+    model_name = model._meta.model_name
+    date = datetime.strptime(model.date, '%Y-%m-%d')
+    path = u'{addon_id}/{date.year}/{date.month:02}/'.format(
+        addon_id=model.addon_id, date=date)
+    name = u'{date.year}{date.month:02}{date.day:02}_{model_name}.json'.format(
+        date=date, model_name=model_name)
+    filepath = os.path.join(path, name)
+    storage.save(filepath, ContentFile(stats_from_model(model)))
```

Comment:
`model` isn't really a model but more an `obj` thus `stats_from_model` could be renamed to `serialize_stats`